lib/metasploit/framework/tcp/client.rb

```diff
@@ -93,7 +93,7 @@ def connect(global = true, opts={})
               'SSLCipher'     =>  opts['SSLCipher'] || ssl_cipher,
               'Proxies'       => proxies,
               'Timeout'       => (opts['ConnectTimeout'] || connection_timeout || 10).to_i,
-              'Context'       => { 'Msf' => framework, 'MsfExploit' => framework_module }
+              'Context'       => { 'Msf' => framework, 'MsfExploit' => self }
```

Comment:
is there any context on this change? 👀

Reply:
Let me revert and re-test and get back to you, this might be fine to revert

Reply:
I assume it was fine to revert? 😄

Reply:
You got it!